onnxruntime/python/session.py

```diff
@@ -192,9 +192,40 @@ def __init__(self, path_or_bytes, sess_options=None, providers=None, provider_op
         self._enable_fallback = True
         self._read_config_from_model = os.environ.get('ORT_LOAD_CONFIG_FROM_MODEL') == '1'
 
-        self._create_inference_session(providers, provider_options)
+        try:
+            self._create_inference_session(providers, provider_options)
+        except:
+            if self._enable_fallback:
```

Comment:
it would be good to print some message indicating there was a fallback and which providers we fell back to.

Reply:
sure, np